<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DroneshareUploadDialog</class>
 <widget class="QDialog" name="DroneshareUploadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>92</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>92</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Droneshare Upload</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="statusLabel">
         <property name="minimumSize">
          <size>
           <width>311</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>upload status</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="uploadProgressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="uploadButton">
         <property name="maximumSize">
          <size>
           <width>91</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>upload</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string>close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
